PicPi/photosUpload.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  photosUpload.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/16/20.
+//
+
+import Foundation
+import Moya
+
+/// photosUpload endpoint
+enum photosUpload {
+  ///enter nameAndExtension as one entry because when using PHAsset will return full file name with extension 
+  case uploadPhoto(photoData: Data , nameAndExtension: String)
```

Comment:
Good comment, wdyt of adding a third value for the extension like so:

```suggestion
  case uploadPhoto(photoData: Data , filename: String, extension: String)
```

Reply:
filename in PHAsset will return the whole thing : filename + extension , this is why I did not want to add another argument